doc/source/install/cmake_options.rst

```diff
@@ -0,0 +1,89 @@
+.. -*- rst -*-
+
+CMake options
+=============
+
+This section describes important options of CMake.
+
+To get more details about installing Groonga from the source with CMake on a specific environment,
+find the document for the specific environment from :doc:`/install`.
+
+``-G GENERATOR``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify a generator.
+
+This is a mandatory option.
+
+You can check available generators by ``cmake --help``.
+
+Here is an example of specifying ``Visual Studio 12 2013 Win64`` as a generator::
+
+  > cmake . -G "Visual Studio 12 2013 Win64" 
+
+``-DCMAKE_INSTALL_PREFIX``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify a directory/folder to install Groonga.
```

Comment:
I think that we can always say "directory".

Reply:
Fixed.